timetracker/timetracker/urls.py

```diff
@@ -16,11 +16,47 @@
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.http import HttpResponse
+from django.views.decorators.http import require_http_methods
 
+# If there's no name, display a form asking for name.
+# This is the HTML for that form.
+form_page_html = """
+  <html>
+    <body>
+      <p>Hi, what's your name?</p>
 
+      <form method="GET">
```

Comment:
You may have received a pull from a lass with handle Bellana.  She's the friend who I said was coming, but then didn't.  Well I met with her this week and channelled you :-)

Anyway, her homework had a Submit button, and it was only on seeing that, that I realised mine probably should have to.  Although, speaking of not needing to specify a default, if you have a single text entry field, the default action for Enter is to submit.

Thanks for the review, and see you tonight!